components/mosquitto/port/include/mosq_broker.h

```diff
@@ -24,6 +25,7 @@ struct mosq_broker_config {
                                      * You can open the respective docs with this idf.py command:
                                      * `idf.py docs -sp api-reference/protocols/esp_tls.html`
```

Comment:
Comment is referring to esp_tls.

Reply:
yes, the comment belongs to the field above (using `/*!< ` tags)

but the docs on the callback is missing. will add -- thanks for noticing!

Reply:
Added doxy-comment to the callback function